app/routes/upgrades.js

```diff
@@ -1,4 +1,12 @@
 import Route from '@ember/routing/route';
+import { A } from '@ember/array';
 
 export default Route.extend({
+  model() {
+    let upgradeArray = A();
+    for (var u of this.game.upgrades.values()) {
+      upgradeArray.pushObject(u)
+    }
+    return upgradeArray
+  },
```

Comment:
I think this gets evaluated every time we look at the page. We should find a way to make it more static, as this never changes. Maybe we can make it a computed thingy?